stdlib/int32.ml

```diff
@@ -57,7 +57,7 @@ let unsigned_to_int =
   | 32 ->
       let max_int = of_int Stdlib.max_int in
       fun n ->
-        if compare zero n <= 0 && compare n max_int <= 0 then
+        if n >= 0l && n <= max_int then
```

Comment:
Not asking you to change this but I personaly write these kind of bound checks as `0l <= n && n <= max_int`, I find it easier for the reader to parse it as a check for membership of `n` to the range `[0l;max_int]`.